src/stories/Library/recommended-material/RecommendedMaterial.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+import { RecommendedMaterial } from "./RecommendedMaterial";
+import recommendedMaterialArgs from "./RecommendedMaterialArgs";
+
+export default {
+  title: "Library / Recommended Material",
+  component: RecommendedMaterial,
+  decorators: [withDesign],
+  // We disable the isChecked control, since it is not possible to
+  // get the states from the React component to work with Storybook controls.
+  argTypes: {
+    favoriteFill: {
+      control: "boolean",
+      defaultValue: true,
+    },
+  },
```

Comment:
Please reconsider this comment.

With the latest change it does not seem relevant anymore.

Reply:
Removed it!